spec/services/workflow_reporter_spec.rb

```diff
@@ -4,44 +4,76 @@
 
 RSpec.describe WorkflowReporter do
   let(:druid) { 'jj925bx9565' }
+  let(:version) { '1' }
+  let(:err_msg) { "Failed to retrieve response #{Settings.workflow_services_url}/preservationAuditWF/something (HTTP status 404)" }
+  let(:wf_server_response_json) { { some: 'json response from wf server' } }
+
+  before do
+    allow(Dor::Workflow::Client).to receive(:new).and_return(stub_wf_client)
+  end
 
   describe '.report_error' do
-    before do
-      allow(Dor::Workflow::Client).to receive(:new).and_return(stub_client)
-    end
+    let(:process_name) { 'moab-valid' }
+    let(:audit_result) { 'Invalid moab, validation error...ential version directories.' }
 
-    let(:stub_client) { instance_double(Dor::Workflow::Client, update_error_status: true) }
+    context 'when workflow already exists' do
+      let(:stub_wf_client) { instance_double(Dor::Workflow::Client, update_error_status: wf_server_response_json) }
 
-    let(:process_name) { 'moab-valid' }
+      it 'returns json response from wf server (mocked here)' do
+        expect(described_class.report_error(druid, version, process_name, audit_result)).to eq wf_server_response_json
+        expect(stub_wf_client).to have_received(:update_error_status)
+          .with(druid: "druid:#{druid}",
+                workflow: 'preservationAuditWF',
+                process: process_name,
+                error_msg: audit_result)
+      end
+    end
 
-    it 'returns true' do
-      result = 'Invalid moab, validation error...ential version directories.'
-      # because we always get true the from the dor-workflow-service gem
-      # see issue sul-dlss/dor-workflow-client#50 for more context
```

Comment:
this issue was addressed, but since it was mocked here, we noticed no change in the specs.